vavr/src/test/java/io/vavr/collection/PriorityQueueTest.java

```diff
@@ -328,4 +329,14 @@ private void assertMinimumsAreEqual(java.util.PriorityQueue<Integer> oldQueue, P
             assertThat(oldQueue.peek()).isEqualTo(newQueue.head());
         }
     }
+
+    // -- ignored tests
+
+    @Override
+    @Test
+    @Ignore
+    public void shouldCalculateAverageOfDoubleAndFloat() {
+        // it is not possible to create a PriorityQueue containing unrelated types
```

Comment:
If your priority queue can be constructed with a customer comparator, it can, right?

Reply:
That's right, will do that...